### PR TITLE
docs(open_graph): update options

### PR DESCRIPTION
**Pull request by SukkaW**
## Check List

**Please read and check followings before submitting a PR.**

- [ ] I want to publish my theme on Hexo official website.
    - [ ] I have read the [theme publishing doc](https://hexo.io/docs/themes#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
    - [ ] `preview` URL is correct.
    - [ ] `preview` URL web site is rendered correctly.
    - [ ] Add a screenshot to `source/themes/screenshots`.
    - [ ] Screenshot filename is same as value of `name`.
    - [ ] Screenshot size is `800 * 500`.
    - [ ] Screenshot file format is `png`.
- [ ] I want to publish my plugin on Hexo official website.
    - [ ] I have read the [plugin publishing doc](https://hexo.io/docs/plugins#Publishing).
    - [ ] `name` is unique.
    - [ ] `link` URL is correct.
- [x] Others (Update, fix, translation, etc...)

Update options supported in `open_graph()`. Will add to other languages after reviewed.

https://github.com/hexojs/hexo/pull/3808 is also included.

### CODE REVIEW

File: source/docs/helpers.md [hunk 1]
```diff
@@ -790,6 +790,13 @@ Option | Description | Default
 `google_plus` | Google+ profile link |
 `fb_admins` | Facebook admin ID |
 `fb_app_id` | Facebook App ID |
+`date` | Article published date | `page.date` if existed
+`updated` | Article modified time | `page.updated` if existed
+`language` | Article language | `page.lang` or `config.language`
```
Comment (curbengh) [inline source/docs/helpers.md]: `if exists`

File: source/docs/helpers.md [hunk 2]
```diff
@@ -790,6 +790,13 @@ Option | Description | Default
...
+
+{% note warn %}
+`og:locale` tag won't be added if given `language` is not in 'language_TERRITORY' format. [`og:locale` documents](https://ogp.me/#optional).
```
Comment (curbengh) [inline source/docs/helpers.md]: an example would make it clearer,

```
...if the `language` value is not in 'language_TERRITORY' format (e.g. 'en_GB'). [`og:local` documentation]...
```

File: source/docs/helpers.md [hunk 3]
```diff
@@ -790,6 +790,13 @@ Option | Description | Default
 `google_plus` | Google+ profile link |
 `fb_admins` | Facebook admin ID |
 `fb_app_id` | Facebook App ID |
+`date` | Article published date | `page.date` if existed
+`updated` | Article modified time | `page.updated` if existed
```
Comment (curbengh) [inline source/docs/helpers.md]: if exists

**Comment by SukkaW** [comment]
@curbengh Rebased and update for all the languages. It is now ready to go.

**Comment by stevenjoezhang** [comment]
Conflict with https://github.com/hexojs/site/pull/1731

**Pull request closed without merging**